release/charts/arguments.go

```diff
@@ -0,0 +1,145 @@
+package charts
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/rancher/ecm-distro-tools/cmd/release/config"
+)
+
+type status struct {
+	ToRelease     map[string][]asset `json:"to_be_released"`
+	ToForwardPort map[string][]asset `json:"to_be_forward_ported"`
+}
+
+type asset struct {
+	Version string `json:"version"`
+}
+
+// loadState will load the lifecycle-status state from an existing state.json file at charts repo
+func loadState(filePath string) (*status, error) {
+	s := &status{}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+
+	data, err := io.ReadAll(file)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := json.Unmarshal(data, &s); err != nil {
+		return nil, err
+	}
+
+	return s, nil
```

Comment:
```suggestion
	var s status

	file, err := os.Open(filePath)
	if err != nil {
		return nil, err
	}

	if err := json.NewDecoder(file).Decode(&s); err != nil {
		return nil, err
	}

	return &s, nil
```
To avoid the ReadAll step, please test before using this

Reply:
@tashima42 

Just tested it,
It works!